examples/pyright/workspace/tests/test_solution.py

```diff
@@ -0,0 +1,10 @@
+# tests/test_solution.py
+import unittest
+
+# TODO Fix `solution` not resolved
+from solution import add
```

Comment:
I can't figure out how to make this resolved. It resolves fine in VS Code using pyright.

`pyrightconfig.json` is documented in https://github.com/microsoft/pyright/blob/main/docs/configuration.md